src/monacoPatch.ts

```diff
@@ -7,6 +7,34 @@ import * as XState from 'xstate';
 import * as XStateModel from 'xstate/lib/model';
 import * as XStateActions from 'xstate/lib/actions';
 
+// dont hate the player, hate the game
+// https://github.com/microsoft/vscode-loader/issues/33
```

Comment:
the issue should get fixed in the future on the Monaco side so we should be able to remove this beautiful hack, yay 🎉